src/com/sap/piper/GitUtils.groovy

```diff
@@ -33,25 +33,22 @@ String getGitCommitId() {
     return sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
 }
 
-String[] extractLogLines(String filter = '',
-                         String from = 'origin/master',
-                         String to = 'HEAD',
-                         String format = '%b') {
+String[] extractLogLines(String filter = '', String from = 'origin/master', String to = 'HEAD', String format = '%b') {
```

Comment:
[style] I prefere the one parameter per line approach over the way choosen here.
The git diff is easier to read with having one parameter per line (e.g. think about adding one parameter. Having one parameter per line we get a new line, otherwise the new parameter is somehow highlighted in a long line marked as diff, when making the diff on bash level the real difference might not be highlighted at all.

Beside that: a long signature in one line leads also to code climate issues if the line gets too long

Reply:
Changed to the way it's done for other methods in this PR.